frontend-react/src/hooks/UseLookupTable.ts

```diff
@@ -139,4 +135,13 @@ const useLookupTable = <T>(
     return valueSetArray;
 };
 
+/* Keeping the dataSetName in here would keep this from refreshing if it changes. */
+export const useValueSetsTable = (dataSetName: string = "") =>
```

Comment:
You might run into issues testing because this empty string exists. I left this in because I remember somewhere you were trying to remove that from the render, too. This should achieve that if you put the right dataSetName in there as a default.